datafusion-expr/src/expr.rs

```diff
@@ -228,6 +228,9 @@ pub enum Expr {
     },
     /// Represents a reference to all fields in a schema.
     Wildcard,
+    QualifiedWildcard {
```

Comment:
It will be better to add extra annotation for `QualifiedWildcard`

Reply:
Do you mean

```rust
Wildcard {
  qualifier: Option<String>
}
```
?

Reply:
No, just add comments to the QualifiedWildcard that distinguish it from the Wildcard